templates/todo/projects/nodejs-mongo-aca/.repo/bicep/infra/main.bicep

```diff
@@ -60,6 +60,13 @@ module containerApps '../../../../../../common/infra/bicep/core/host/container-a
     tags: tags
     containerAppsEnvironmentName: !empty(containerAppsEnvironmentName) ? containerAppsEnvironmentName : '${abbrs.appManagedEnvironments}${resourceToken}'
     containerRegistryName: !empty(containerRegistryName) ? containerRegistryName : '${abbrs.containerRegistryRegistries}${resourceToken}'
+    // Work around Azure/azure-dev#3157 (the root cause of which is Azure/acr#723) by explicitly enabling the admin user to allow users which 
+    // don't have the `Owner` role granted (and instead are classic administrators) to access the registry to push even if AAD authentication fails.
+    //
+    // This addresses the following error during deploy:
+    //
+    // failed getting ACR token: POST https://<some-random-name>.azurecr.io/oauth2/exchange 401 Unauthorized
+    containerRegistryAdminUserEnabled: true
```

Comment:
There are 3 `aca` projects.  I think we need the fix on all of them?

Reply:
I will look into this.  Thanks for the call out!

Reply:
Thanks - fixed this in the other two (and did a search for "container-apps.bicep" to confirm that these were the only things using it).